doc/Public_API.md

```diff
@@ -28,7 +28,7 @@ healthcheck:
 livenessProbe:
     httpGet:
         path: /platform/api/modules/health/live
-        port: http
+        port: http # Adapt based on PLATFORM_LISTENING_PORT
```

Comment:
should be defined in container port 
we can maybe add this for completness
```yaml
        ports:
          - containerPort: 8080
            protocol: TCP
            name: http 
         env: 
          - name: CDK_LISTENING_PORT
            value: 8080
```

Reply:
Right, I did not want to "override" `http` assuming it defaults to 80 when used (for K8s that is).

I suppose, either way, the healtcheck needs to point to the port defined by `CDK_LISTENING_PORT`